src/mock/subscription.ts

```
@@ -0,0 +1,125 @@
+export const MockUserSubscriptions: UserSubscription[] = [
+  {
+    id: "sub_001",
+    productId: "p_ai_001", // ChatGPT Plus
+    planId: "pl_ai_001",
+    startDate: new Date("2024-11-01"),
+    endDate: new Date("2025-06-15"), // 약 6개월 후
+    status: "active",
+    isParty: true,
+    partyId: "party_001",
+    currentMembers: 3,
+    maxMembers: 4
```

ESLint

[error] Insert `,`

(prettier/prettier)